ddtrace/span.py

```diff
@@ -132,6 +132,10 @@ def set_tag(self, key, value):
         except Exception:
             log.debug("error setting tag %s, ignoring it", key, exc_info=True)
 
+    def remove_tag(self, key):
```

Comment:
These new methods will change our public API (so it can't be considered a bugfix). I would say to make them internal with an underscore. This way, we can use them without exposing to the public:
* `_remove_tag`
* `_remove_traceback`
* `_remove_exc_info`